spec/models/photo_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+
+RSpec.describe Photo, type: :model do
+
+  it { is_expected.to belong_to :user }
+  it { is_expected.to have_many :comments }
+  it { is_expected.to have_many :likes }
+
+  describe 'build_with_user' do
+
+    let(:user) { create(:user) }
+
+    let(:photo) { Photo.create title: "test",
+    image:  Rack::Test::UploadedFile.new(Rails.root + 'spec/assets_specs/photos/test_photo1.jpg', 'image/jpg')}
+    let(:comment_params) { { comment: 'text' } }
+
+    subject(:comment) { photo.comments.build_with_user(comment_params, user) }
+
+    it 'builds a comment' do
+      expect(comment).to be_a Comment
+    end
+
+    it 'builds a comment associated with the specific user' do
+      expect(comment.user).to eq user
+    end
+
+  end
+
+
+  describe 'create_with_user' do
+
+    let(:user) { create(:user) }
+
+    let(:photo) { Photo.create title: "test",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.